pkg/cidr/cidr.go

```diff
@@ -272,3 +272,38 @@ func uintToIP(ip uint32) net.IP {
 
 	return netIP
 }
+
+func GetValidClusterSize(cidrRange string, clusterSize uint) (uint, error) {
```

Comment:
I think the term "ClusterSize" is a bit confusing. Elsewhere in this code, I referred to it as "AllocationSize".

```suggestion
func GetValidAllocationSize(cidrRange string, allocationSize uint) (uint, error) {
```

Reply:
Agree. I wasn't too happy with it, so left it similar to globalnet to discuss during review.